internal/nix/nix.go

```diff
@@ -144,3 +149,19 @@ func System() (string, error) {
 func ProfileBinPath(projectDir string) string {
 	return filepath.Join(projectDir, ProfilePath, "bin")
 }
+
+func isExitErrorInsecurePackage(err error) (bool, error) {
+	var exitErr *exec.ExitError
+	if errors.As(err, &exitErr) && exitErr.ExitCode() == 1 {
+		if strings.Contains(string(exitErr.Stderr), "is marked as insecure") {
```

Comment:
😭

Reply:
yeah this is not the best. An alternative is to loop though all packages and check if any of them are insecure, that would make errors much slower so it's a tradeoff.